draft-ietf-tls-ticketrequests.md

```diff
@@ -153,6 +153,15 @@ IANA is requested to Create an entry, ticket_request(TBD), in the existing regis
 for ExtensionType (defined in {{RFC8446}}), with "TLS 1.3" column values being set to
 "CH", and "Recommended" column being set to "Yes".
 
+# Performance Considerations
+
+Servers can send the requested tickets in NewSessionTicket messages at any time after the
+client Finished message. If the server chooses to send a large number of tickets to the client,
+it can potentially harm application performance if the tickets are sent before application data.
```

Comment:
```suggestion
can potentially harm application performance if the tickets are sent before application data.
```

Reply:
I don't think removing the word works, grammatically?

Reply:
If you apply them all it should :)

Reply:
Ah the other part had rolled off